ddl/job_table.go

```diff
@@ -661,6 +661,21 @@ func GetBackfillJobCount(sess *session, tblName, condition string, label string)
 	return int(rows[0].GetInt64(0)), nil
 }
 
+func getUnsyncedInstanceIDs(sess *session, jobID int64, label string) ([]string, error) {
```

Comment:
Do we need to know the details of these instances? The only usage of this function is to check the length: `len(unsyncedInstanceIDs) == 0`.

Maybe this could be improve the SQL as `select count(1) from mysql.tidb_ddl_backfill_history where ddl_job_id = %d and state != ${model.JobStateSynced}`.

Reply:
So far, I've only set the synced state of one backfill job per instance, so I use this statement